pkg/volume/vsphere_volume/vsphere_volume.go

```diff
@@ -75,22 +74,22 @@ func (plugin *vsphereVolumePlugin) NewUnmounter(volName string, podUID types.UID
 	return plugin.newUnmounterInternal(volName, podUID, &VsphereDiskUtil{}, plugin.host.GetMounter())
 }
 
-func (plugin *vsphereVolumePlugin) newMounterInternal(spec *volume.Spec, podUID types.UID, manager cdManager, mounter mount.Interface) (volume.Mounter, error) {
+func (plugin *vsphereVolumePlugin) newMounterInternal(spec *volume.Spec, podUID types.UID, manager vdManager, mounter mount.Interface) (volume.Mounter, error) {
 	var vvol *api.VsphereVirtualDiskVolumeSource
```

Comment:
vdVol or vVol.  If we are using the 'vd' convention for the manager we should stick to the same pattern